study-builder/studies/osteo/emails/reconsent_next_steps_2wk_reminder.html

```diff
@@ -0,0 +1,120 @@
+       border="0" cellpadding="0" cellspacing="0" width="100%">
+  <tr style="width: 540px;">
+    <td align="left" style="margin: 0; padding: 50px 0 0 0; font-family: 'Source Sans Pro', sans-serif;">
+      <a href="-ddp.baseWebUrl-" target="_blank" style="cursor: pointer; text-decoration: none;">
+        <img src="https://storage.googleapis.com/cmi-study-dev-assets/osteo/project-logo.png"
+             alt="Osteosarcoma Project logo" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;"
+             width="230px" height="63px" border="0">
+      </a>
+    </td>
+  </tr>
+
+  <tr style="width: 540px;">
+    <td style="padding: 50px 0 0 0;">
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.participant.firstName-,</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Thank you for being part of the Osteosarcoma Project. Two weeks ago, we sent you a link to sign a consent form in order to continue participation in the project. We are resending the link in case it was missed.</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Click on the link below in order to follow a short process where you may:</p>
+
+      <ol>
+        <li style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Make an account to access all your information in the Osteo Project</li>
+        <li style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Opt-in to a new consent form for ongoing participation in the Osteo Project</li>
+        <li style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Provide any updates to your medical history, including new institutions or physicians you’ve seen</li>
+      </ol>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">This process should take about 15-20 minutes.</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="-ddp.baseWebUrl-/activity-link/-ddp.activityInstanceGuid-" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```

Comment:
Since we need child to sign up for an account before filling out the consent form, the URL should point to `-ddp.baseWebUrl-/accept...`.